### DISCUSSION TITLE
New Loss Functions or Custom Loss Functions?

### DISCUSSION THREAD

**Discussion by RobotPsychologist** [amazon-science/chronos-forecasting#480] [Ideas]
Is there a way to add new customized loss metrics to the repository for fine-tuning? We have some use cases we would like to explore for some unique loss functions, happy to contribute this to the repository myself if there's an easy way to do that.
